Resources/Locale/en-US/prototypes/catalog/fills/crates/fun-crates.ftl

```diff
@@ -51,3 +51,6 @@ ent-CrateFunBikeHornImplants = Bike Horn Implants
 
 ent-CrateFunMysteryFigurines = Mystery Figure Crate
     .desc = A collection of 10 Mystery Figurine boxes. Duplicates non refundable.
+	
+ent-CrateFunDartsSet = Dartboard Box Set
+    .desc = A box with everything you need for a fun game of darts.
```

Comment:
hupodart is already built into the PR, it is bought in uplink for 3 TC

I'm not sure, then it will be the only crate that will have its own description and name. I did it in the image and likeness of a box of water pistols, and it has a name and description (like all the boxes purchased in cargo) done the same way - there is no own description and name, and it is done via ftl